.travis.yml

```diff
@@ -9,6 +9,7 @@ services:
 
 script:
   - docker build -t theconnman/docker-hub-rss .
+  - docker build -t theconnman/docker-hub-rss -f Dockerfile-armv7 .
 
 after_success:
```

Comment:
You also need to push the new image you've built to Docker hub in this section.

Reply:
How would I do that? Do I do `docker push theconnman/docker-hub-rss:latest-armv7`?

Reply:
Exactly.